eval/Containerfile

```diff
@@ -0,0 +1,21 @@
+FROM nvcr.io/nvidia/cuda:12.4.1-devel-ubi9
+
+RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc gcc-c++ \
+    && dnf clean all \
+    && rm -rf /var/cache/*dnf*
+
+WORKDIR /instructlab
+
+COPY eval/requirements.txt .
+RUN python3.11 -m ensurepip
+RUN python3.11 -m pip install --no-cache-dir git+https://github.com/instructlab/eval && python3.11 -m pip install -r requirements.txt
+
+RUN export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/python3.11/site-packages/nvidia/cudnn/lib"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/python3.11/site-packages/nvidia/cudnn/lib"
+ENV HOME=/instructlab
+
+RUN chmod -R 777 /instructlab
+USER 1001
+COPY eval/eval-scripts .
+
+ENTRYPOINT ["/bin/bash"]
```

Comment:
Set this to

```
CMD ["python"]
```

See my PR for details: https://github.com/redhat-et/ilab-on-ocp/pull/2/files#diff-602239348cc944737a4fc15b5d1b8509bf60ddacf0152b5027e4db16916312acR4

Then you can turn your `@dsl.container_components` into `@dsl.components` and run your python directly - no need for shell -> python -> script overhead.

Reply:
I wonder if I use `CMD ["python3.11"` instead of `ENTRYPOINT ["3.11"]` dsl will indeed run 3.11 and not 3.9. From my empirical data, dsl runs 3.9 in this container not 3.11. I had to explicity set 3.11 in the generated yaml 🤷‍♀️ but yes, I've updated to use component rather than the container->shell->python

Reply:
https://github.com/redhat-et/ilab-on-ocp/pull/6/commits/2c2bd58ae03137ed6d3b6998246fd8f67825110d